File: Practica-CodigosPython/Ej11.py
#Dados dos numeros positivos mostrar todos los numeros primos en ese rango de numeros
#Ejemplo: A=2 B=10 2,3,4,7
A=int(input("Intro A: "))
B=int(input("Intro B: "))
for i in range(A,B+1):
    c1=1; c2=0
    while c1<=i:
        if i%c1==0:
            c2=c2+1
        c1=c1+1
    if c2==2:
        print(i,end=", ")

File: Practica-CodigosJava/Ej11.java
//Dados dos numeros positivos mostrar todos los numeros primos en ese rango de numeros
//Ejemplo: A=2 B=10 2,3,4,7
import java.util.Scanner;
public class Ej11 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Intro A: ");
        int A = scanner.nextInt();
        System.out.print("Intro B: ");
        int B = scanner.nextInt();
        for (int i = A; i <= B; i++) {
            int c2 = 0; // Contador de divisores
            for (int c1 = 1; c1 <= i; c1++) {
                if (i % c1 == 0) {
                    c2++; // Incrementar contador si es divisor
                }
            }
            if (c2 == 2) {
                System.out.print(i + ", "); // Imprimir número primo
            }
        }
        scanner.close();
    }
}